docs/source/reference/oauth.md

```diff
@@ -291,10 +291,10 @@ but the information from the _external_ authentication provider
 Authenticator configuration governs when JupyterHub needs to ask again,
 triggering the external login process anew before letting a user proceed.
 
-- `jupyterhub-hub-login` cookie stores that a browser is authenticated with the Hub.
+- `JupyterHub-hub-login` cookie stores that a browser is authenticated with the Hub.
```

Comment:
These cookie names (all replacements below here, I think) are specific strings in the code, and should not be capitalized. have to be careful with find-replace!